protractor/helpers/reporter.ts

```diff
@@ -1,6 +1,13 @@
 import { SpecReporter, StacktraceOption } from 'jasmine-spec-reporter';
+import { AwesomeReport } from 'jasmine-awesome-report';
```

Comment:
On the Jasmine tutorial, they say to use "const { AwesomeReport } = require('jasmine-awesome-report');"
but instead, I imported it using "import { AwesomeReport } from 'jasmine-awesome-report';" in order to be consistent with the TypeScript syntax and do not combine it with pure Javascript.
Anyway, if it is better to import the module the JavaScript why, please tell me and I will change it.

Reply:
As you said, in this case is better to import using the typescript way.